geoips/commandline/args.py

```diff
@@ -560,6 +560,13 @@ def add_args(parser, arglist=None):
             type=str.upper,
         )
 
+    if arglist is None or "no_sectoring" in arglist:
+        procflow_group.add_argument(
+            "--no_sectoring",
```

Comment:
Let's just make sure we are all in agreement on the name of this option before we merge.  I can't think of anything else off the top of my head, so I think no_sectoring is fine, but I'll give someone else an opportunity to chime in :-)

Reply:
Yeah, I think this is fine for now. This will go away with order-based and the new CLI so I'm not too worried about it.